src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlSuggestionRow.xaml

```diff
@@ -0,0 +1,103 @@
+﻿<!--
+Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+-->
+<UserControl x:Class="JuliusSweetland.OptiKey.Models.XmlSuggestionRow"
```

Comment:
I have not been able to apply background overrides to suggestion keys and I still haven't figured out why.

Reply:
Ok, we can come back to this